app/models/spree/order.rb

```diff
@@ -145,6 +146,14 @@ def states
       end
     }
 
+    scope :sort_by_billing_address_name_asc, -> {
```

Comment:
These are defined here rather than on `Spree::Address` because of...

> First, a scope involving child associations needs to be defined in the parent table model, not in the child model. 
https://activerecord-hackery.github.io/ransack/going-further/other-notes/#using-scopesclass-methods